.github/workflows/containerize-agent.yaml

```
@@ -103,8 +103,24 @@ jobs:
         id: get_version
         run: echo "POETRY_VERSION=$(poetry version -s)" >> $GITHUB_OUTPUT
 
+      - name: Get is_release from pyproject.toml
+        id: get_is_release
+        run: echo "IS_RELEASE=$(poetry version -s | grep -q 'dev' && echo false || echo true)" >> $GITHUB_OUTPUT
+
       # Uses the version from pyproject.toml for the tag
-      - name: Push to Docker Hub and Internal ECR
+      - name: Push Pre-Release to Docker Hub and Internal ECR
+        if: ${{ steps.get_is_release.outputs.IS_RELEASE == 'false' }}
+        uses: docker/build-push-action@v5
+        with:
+          context: .
+          platforms: linux/amd64,linux/arm64
+          push: true
+          tags: |
+            greatexpectations/agent:${{ steps.get_version.outputs.POETRY_VERSION }}
+            258143015559.dkr.ecr.us-east-1.amazonaws.com/gx/agent:${{ steps.get_version.outputs.POETRY_VERSION }}
+
+      - name: Push Pre-Release to Docker Hub and Internal ECR
+        if: ${{ steps.get_is_release.outputs.IS_RELEASE == 'false' }}
```

Should this section be for release not pre-release?
```suggestion
      - name: Push Release to Docker Hub and Internal ECR
        if: ${{ steps.get_is_release.outputs.IS_RELEASE == 'true' }}
```